ghost/i18n/locales/hu/comments.json

```diff
@@ -8,13 +8,13 @@
     "Add comment": "Hozzászólás",
     "Add context to your comment, share your name and expertise to foster a healthy discussion.": "Segítsd a beszélgetést azzal, hogy megadod a nevedet és azt, hogy mihez értesz.",
     "Add reply": "Válasz",
-    "Add your expertise": "",
+    "Add your expertise": "Add meg a szakterületedet",
     "Already a member?": "Van már fiókod?",
     "Anonymous": "Anonim",
     "Are you sure?": "Biztos vagy benne?",
-    "Become a member of {publication} to start commenting.": "Hozzászólást csak tagok írhatnak.",
-    "Become a paid member of {publication} to start commenting.": "Hozzászólást csak előfizetők írhatnak.",
-    "Best": "",
+    "Become a member of {publication} to start commenting.": "Csatlakozz a {publication} közösséghez, hogy hozzászólhass a tartalmakhoz.",
+    "Become a paid member of {publication} to start commenting.": "Csatlakozz a fizető {publication} közösséghez, hogy hozzászólhass a tartalmakhoz.",
```

Comment:
AI is complaining that this translation is not idiomatic.  If you imagine a variety of site titles in for {publication}, do you like how it reads?